README.md

```diff
@@ -330,6 +330,17 @@ Then you can run this in the checkout root to reformat all Rust code:
 $ make rustfmt
 ```
 
+### Running tests
+
+* make check
+  Run all tests as defined in the directory.  Expensive tests are
```

Comment:
Which directory is the directory? I'd be tempted to say "Run elisp and Rust tests. If run in a subdirectory, only run the tests in that directory." (assuming I've understood correctly)

Reply:
You're right, I already thought about adding such a note.